app/views/layouts/application.html.haml

```diff
@@ -19,6 +19,9 @@
               = link_to "Invoices", ""
             %li{class: "#{top_navbar_active_class_name('clients','index')}"} 
               = link_to "Clients", clients_path
+            -if current_user.present? && current_user.role.to_s == "Admin"
```

Comment:
move this condition to a helper as a method

```
def is_admin?
  current_user.present? && current_user.role.to_s == "Admin"
end
```
and call this method wherever required